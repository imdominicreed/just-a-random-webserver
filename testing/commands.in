getDeleteRows
getFile no_file
deleteRow no_file
markFileDeleted no_file
insertFile 1 test1
insertFile 2 test2
insertFile 3 test3
getFile test1
getFile test2
getFile test3
insertFile 1 test1
insertFile 2 test1
markFileDeleted test1
markFileDeleted test2
getDeleteRows
deleteRow test1
deleteRow test2
getFile test1
getFile test2
getFile test3
quit

