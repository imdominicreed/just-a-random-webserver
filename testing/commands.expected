Printing rows!
File not Found!
No Deletetion!
No Mark Deletetion!
Inserted!
Inserted!
Not inserted!
id: 1 file name: test1 deleted: False
id: 2 file name: test2 deleted: False
id: 3 file name: test3 deleted: False
Not inserted!
Not inserted!
Mark Deleted!
Mark Deleted!
Printing rows!
id: 1 file name: test1 deleted:  True
id: 2 file name: test2 deleted:  True
Deleted!
Deleted!
File not Found!
File not Found!
id: 3 file name: test3 deleted: False
